virtcontainers/device/drivers/vfio.go

```diff
@@ -97,17 +114,38 @@ func (device *VFIODevice) DeviceType() config.DeviceType {
 	return device.DevType
 }
 
-// getBDF returns the BDF of pci device
-// Expected input strng format is [<domain>]:[<bus>][<slot>].[<func>] eg. 0000:02:10.0
-func getBDF(deviceSysStr string) (string, error) {
-	tokens := strings.Split(deviceSysStr, ":")
+type VFIODevice_Type int32
+
+const (
+        // 0 is reserved for errors.
```

Comment:
I'd still be tempted to add a:

```
const VFIODevice_TYPE_VFIO_ERROR = 0
```

... as you can return that from `getVFIODeviceType()`.

Zero tends to denote success so doing this would make the code clearer.

Reply:
ok, I will change it in the next review.